scripts/monitor-system-ressources.sh

```
@@ -0,0 +1,6 @@
+source "$PROJECT_DIR/scripts/system_monitoring_functions/cpu_stats.sh"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)